src/utils/utils.go

```diff
@@ -287,6 +287,10 @@ func IsStatusFailed(operatorStatus models.OperatorStatus) bool {
 	return operatorStatus == models.OperatorStatusFailed
 }
 
+func IsStatusSucceeded(operatorStatus models.OperatorStatus) bool {
```

Comment:
i think it should be pard of operators handler file

Reply:
Done.